ksp_plugin/pile_up.cpp

```diff
@@ -98,7 +101,9 @@ Status PileUp::DeformAndAdvanceTime(Instant const& t) {
   absl::MutexLock l(lock_.get());
   Status status;
   if (psychohistory_->back().time < t) {
-    DeformPileUpIfNeeded();
+    angular_momentum_ += intrinsic_torque_ * (t - psychohistory_->back().time) +
+                         angular_momentum_change_;
```

Comment:
Could we move this computation to the beginning of `DeformPileUpIfNeeded`?  It's odd that it's floating here for no good reason.

Reply:
That does not make sense.